### ISSUE TITLE
Strategize Web Content Accessibility

### ISSUE THREAD

**Issue by entrotech** [hackforla/food-oasis#325]
### Overview
Little attention has been paid to application accessibility for the Minimum Viable Product phase. A production site should be compliant with WCAG guidelines.

### Action Items

- [ ] Review the application for Accessibility [WCAG Accessbilty Guidelines](https://www.w3.org/WAI/fundamentals/),.
- [ ] Identify appropriate Web Accessibility Checker Tool(s) for the application and run them to identify gaps.
- [ ] Identify how we might include accessiblity checking into our testing, code checking and release processes.
- [ ] Document guidance for designers/developers to implement best practices.
- [ ] Make a presentation to the team with an overview/demonstration of accessbility principles and practices.
- [ ] Create specific  issues for designers and developers to close the accessibility gaps.

**Comment by ExperimentsInHonesty** [comment]
### Overview
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

**Comment by Benbaillou** [comment]
Hi - can we label the size please?

**Comment by staceyrebekahscott** [comment]
Would like to consider whether we should do this before or after we do user research. I imagine some things identified by users may also be accessibility issues, but I think it might be beneficial for prioritizing if we get user feedback on the website as it is now. This way we can build the features that our particular user base needs first, rather than relying on a general set of accessibility guidelines.

**Comment by fancyham** [comment]
- Related to #748
